invenio_app_ils/ui/src/semantic-ui/site/elements/container.overrides

```diff
@@ -20,80 +35,137 @@
 
-  .ui.container.footer-lower {
-    background-color: #2A2D34;
-    padding-top: @footerPaddings/2;
-    padding-bottom: @footerPaddings/2;
-    color: white;
-  }
+    &.dot-background-container {
+      position: relative;
+    }
 
-  .ui.container.footer-upper {
-    background: rgb(52, 55, 62);
-    background: linear-gradient(155deg, rgba(52, 55, 62, 1) 0%, rgba(71, 75, 85, 1) 100%);
-    padding-top: @footerPaddings;
-    padding-bottom: @footerPaddings;
-    color: #c1bbbb;
-  }
+    &.dot-background {
+      &:after {
+        content: "";
+        background: linear-gradient(90deg, white (@dot-bg-space - @dot-bg-size) 50%, transparent 1%) center,
+        linear-gradient(white (@dot-bg-space - @dot-bg-size), transparent 1%) center,
+        @dot-bg-color;
+        background-size: @dot-bg-space @dot-bg-space;
+        top: 50%;
+        left: 0;
+        width: 100%;
+        height: 65%;
+        position: absolute;
+        overflow: hidden;
+        z-index: -1;
+      }
+    }
 
-  /* Highlighted sections */
+    /* first section moved up */
 
-  .ui.container.section {
-    background-color: @highlightBackgroundColor;
-    padding-top: 1em;
-    padding-bottom: 3em;
-    border-bottom: 1px solid #c6c4d0;
-    border-top: 1px solid #c6c4d0;
-    position: relative;
-  }
+    &.fs-landing-page-section:first-child {
+      background: rgb(255, 255, 255);
+      background: linear-gradient(0deg, rgba(255, 255, 255, 0) 50%, rgba(255, 255, 255, 1) 65%);
+      margin-bottom: 10em;
+      margin-top: -4em;
+    }
 
-  /* search pages */
+    &.fs-landing-page-section {
+      margin-bottom: 3rem;
+    }
 
-  .ui.container.search-body {
-    padding: @sectionDefaultPadding;
-  }
+    /* Frontsite search bar container */
+
+    &.document-details-search-container {
+      background-color: @highlightBackgroundColor;
+      padding-top: @defaultPaddingValue;
+      padding-bottom: @defaultPaddingValue;
+      border-bottom: 1px solid #c6c4d0;
+      position: relative;
+
+      &:before, &:after {
+        content: "";
+        position: absolute;
+        z-index: -1;
+        box-shadow: 0 0 20px -8px rgba(0, 0, 0, 0.2);
+        top: 50%;
+        bottom: 0;
+        left: 43%;
+        right: 43%;
+        border-radius: 100px / 10px;
+      }
+    }
+
+    /* frontsite footer */
+
+    &.footer-lower {
+      background-color: #2A2D34;
+      padding-top: @footerPaddings/2;
+      padding-bottom: @footerPaddings/2;
+      color: white;
+    }
+
+    &.footer-upper {
+      background: rgb(52, 55, 62);
+      background: linear-gradient(155deg, rgba(52, 55, 62, 1) 0%, rgba(71, 75, 85, 1) 100%);
+      padding-top: @footerPaddings;
+      padding-bottom: @footerPaddings;
+      color: #c1bbbb;
+    }
+
+    /* Highlighted sections */
+
+    &.section {
+      background-color: @highlightBackgroundColor;
+      padding-top: 1em;
+      padding-bottom: 3em;
+      border-bottom: 1px solid #c6c4d0;
```

Comment:
This `borderColor` can also be a variable.

Reply:
done